roles/foreman_repositories/molecule/debian/verify.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Verify
+  hosts: all
+  gather_facts: false
+  tasks:
+    - name:
+      stat:
```

Comment:
do we *need* the `stat` here? Or shall we rely on `slurp` later failing if it's not there?